source/_components/xbox_live.markdown

```diff
@@ -43,4 +47,8 @@ xuid:
   description: Array of profile XUIDs to be tracked.
   required: true
   type: list
+scan_interval:
```

Comment:
`scan_interval` should not be documented on the platform, since it is a platform option.

Reply:
Got it, wasn't sure, so I erred on the side of "include". I'll remove the extra.